MyStore/MyStore.cpp

```diff
@@ -1,7 +1,16 @@
-// MyStore.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
-int main()
-{
-	// Add some code here
-}
+﻿#include "JsonParcer.h"
+
+
+int main()
+{
+	JsonParcer obj(R"(D:\Projects\PeexProject\Data.json)");

```

Comment:
- Why do you need R prefix here? It seems redundant.
- Please use relative path (e.g. "..\data\Data.json") to avoid binding to exact path. Note that repository can be cloned to different path than on your system and it still should work.
- Add this Data.json to repository, for instance as \data\Data.json, where \data\ folder is on the same level as \src\. Without this file it's impossible to test program in runtime.